pkgs/by-name/in/intel-compute-runtime/package.nix

```diff
@@ -69,16 +71,21 @@ stdenv.mkDerivation rec {
       $out/lib/intel-opencl/libigdrcl.so
   '';
 
+  passthru.updateScript = nix-update-script { };
```

Comment:
This will generate some false positives as we only update to releases and never to tags.

Reply:
Is there a different script that uses releases?

Reply:
When I ran it locally, it suggested a downgrade to me. Please just omit the updater for now. We also need to check the dependencies for each version, so auto updates are kinda dangerous.